src/backend/web/routes/admin.js

```diff
@@ -1,8 +1,26 @@
+require('../../lib/config');
 const express = require('express');
 const { UI } = require('bull-board');
+const fs = require('fs');
 
 const router = express.Router();
 
 router.use('/queues', UI);
 
+router.get('/log', (req, res) => {
+  if (!process.env.LOG_FILE) {
+    res.send('LOG_FILE undefined in .env file');
+  } else {
+    try {
+      console.log("I'm trying");
+      const readStream = fs.createReadStream(process.env.LOG_FILE, 'utf8');
+
+      res.append('Content-type', 'text/plain');
+      readStream.on('data').pipe(res);
```

Comment:
You should be able to pipe your readStream directly to res:

```
readStream.pipe(res)
```

After this, you should probably add `.on('error')` event handlers for both the `readStream` and also `res`, and in both cases, log the error and close the streams.